src/components/Tooltip.svelte

```diff
@@ -0,0 +1,114 @@
+<script>
+  // parent block has to have position: relative styles
+  import { fly, fade } from 'svelte/transition';
+  import { circIn } from 'svelte/easing';
+  import { onMount } from 'svelte';
+
+  export let text = '';
+
+  const defaultTooltipInset = {
+    left: '50%',
+    right: 'auto',
+    translateX: '-50%',
+    bottom: 0
+  };
+
+  const commonAnimationSettings = { duration: 300 };
+  const flyAnimationSettings = { ...commonAnimationSettings, y: -5 };
+
+  let tooltipExists = false;
+  let isTooltipVisible = false;
+  let tooltipInset = defaultTooltipInset;
+  let parentRef = null;
+  let tooltipRef = null;
+
+  onMount(() => {
+    if (parentRef.children.length > 1) {
```

Comment:
if it expects only one child and will not work without it, you can use `length !== 1` and rephrase error message